docs/book/src/roadmap.md

```diff
@@ -1,21 +1,27 @@
+CAPZ is the official production-ready Cluster API implementation to administer the entire lifecycle of self-managed or managed Kubernetes clusters (AKS) on Azure. Cluster API extends the Kubernetes API to provide tooling consistent across on-premises and cloud providers to build and maintain Kubernetes clusters at scale while working with GitOps and the surrounding tooling ecosystem. [See related blog post.](https://cloudblogs.microsoft.com/opensource/2023/04/20/kubernetes-at-scale-with-gitops-and-cluster-api/)
 
 ## Epics
 
-There are a number of large priority "Epics" which may span across milestones which we believe are important to providing CAPZ users an even better experience and improving the vision.  
+There are a number of large priority "Epics" which may span across milestones which we believe are important to providing CAPZ users an even better experience and improving the vision.  The [CAPZ project board roadmap view](https://github.com/orgs/kubernetes-sigs/projects/26/views/11) tracks the larger "epic issues" and their progress.
 
--  Latest Core Dependencies - The foundation should keep everything under a supported version of dependencies as well as to enable the latest features. 
-    - Includes: [track 2 go-sdk](https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/2670), [Azure Service Operator (ASO)](https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/416), 100% [k8s out-of-tree Azure provider](https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/715)
--  ManagedClusters - Provisioning new AKS (ManagedClusters) clusters at scale is a common use case we want to enable an excellent experience with stability and all of the same features available in AKS.
-    - Includes: [ManagedClusters E2E Tests](https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/2873), [ClusterClass support](https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/2684), [Enabling all AKS features](https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/2625), [Evolution of standardized CAPI ManagedCluster for CAPZ](https://github.com/kubernetes-sigs/cluster-api/blob/main/docs/proposals/20220725-managed-kubernetes.md)
--  Latest features for self-managed - These larger or essential features enable a better experience for provisioning self-managed clusters on Azure.
-    - Includes: MachinePools support, Azure CNI v2 support
+- ManagedClusters - Provisioning new AKS (ManagedClusters) clusters at scale is a common use case we want to enable an excellent experience with stability and all of the same features available in AKS. Includes:
+  - [ClusterClass support](https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/2684)
+  - [Enabling all AKS features](https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/3446)
```

Comment:
Wonder if enabling preview features (https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/2625) should get its own bullet point?